web_monitoring/tests/test_cli.py

```diff
@@ -48,6 +49,20 @@ def test_filter_unchanged_versions():
     ]
 
 
+def test_is_valid():
+    valid_single_url = ['https://example.com']
+    valid_urls = ['https://example.com', 'http://example.com/interesting/path', 'https://subdomain.example.com']
+    valid_url_missing_domain = ['https://google']
+    invalid_single_url = ['asldkfje']
+    invalid_and_valid_urls = ['https://exmaple.com', 'http:badprotocol.com']
+
+    assert all(_is_valid(url) for url in valid_single_url)
+    assert all(_is_valid(url) for url in valid_urls)
+    assert all(_is_valid(url) for url in valid_url_missing_domain)
+    assert not all(_is_valid(url) for url in invalid_single_url)
+    assert not all(_is_valid(url) for url in invalid_and_valid_urls)
```

Comment:
Now that this function isn’t working on a list, I’m not sure it makes sense to be testing lists of things here. It’s fine as-is, but would be nicer if these were just:

```py
assert _is_valid(some_valid_url)
assert not _is_valid(some_invalid_url)
# etc.
```

Reply:
Ah yeah, you're right. That's a quick fix, let me tidy up before we merge.